packages/neutrino-middleware-image-loader/index.js

```diff
@@ -3,18 +3,17 @@ const merge = require('deepmerge');
 module.exports = ({ config }, options = {}) => {
   const { limit } = merge({ limit: 8192 }, options);
```

Comment:
I'm not sure if this is expected (ie to prevent the presets being overwhelmed with options), but none of the presets actually pass in any options to this or some of the other middleware - so this functionality is only being used by those who call the middleware manually.

Reply:
Yeah, we don't use this internally, this is just for consumers of the middleware who happen to want to override options.